CryptoBinPkg/Driver/Crypto.c

```diff
@@ -517,6 +517,11 @@ const EDKII_CRYPTO_PROTOCOL  mEdkiiCrypto = {
  #else
   NULL,
  #endif
+ #if _PCD_VALUE_PcdCryptoServiceX509CompareDateTime

```

Comment:
Can you move this to the end file right before the NULL entry?  That will make the changes backwards compatible.